### ISSUE TITLE
Issues at 200% zoom

### ISSUE THREAD

**Issue by singhinderdeep** [bcgov/digital.gov.bc.ca#272]
Ideally, the page should be totally functional even when magnified at 200%.

**Problem**
Seems to be a container width issue as some visual inconsistencies are visible like hero section, background color and footer. Please refer to the attached image fro clarification.!

[Screen Shot 2020-08-28 at 4.01.42 PM.png](https://images.zenhubusercontent.com/5eb1dbd03cd5665e2aefe526/43de7978-ed97-4e82-b388-8e1fc3e96936)

**Comment by HeatherRemacle** [comment]
This is way improved, and the content generally renders visibly. A few small issues.

1) Home page header text cut off on the right:
![image.png](https://images.zenhubusercontent.com/5eb1cfe9cf91c115cba78411/56250579-6b2b-4072-9378-f73a306ea509)

2) Products and services page 4 card spread at the top cuts off slightly on the left (only spot on the page that does that):
![image.png](https://images.zenhubusercontent.com/5eb1cfe9cf91c115cba78411/af5d53c5-febd-4505-b4da-d499020bb5c8)

I was viewing this deployment for reference: https://digital-gov-frontend-pr-348-xdwiwd-dev.pathfinder.gov.bc.ca/products-services 

I'm happy to close this ticket, as the content generally renders properly, and have the two issues above as very minor bug fixes/good first issues in our backlog.

**Comment by mark-a-wilson** [comment]
I'd say let's close this issue as the original problem that Inderdeep pointed out with his .png has been resolved.

Heather, the additional issues you point out here are valid, so I will create two new, well-defined Issues to resolve those.

**Comment by mark-a-wilson** [comment]
Confirmed at standup this morning we can close this issue. New issues identified will be created.
